scripts/2022-06-07_secret_pepper_dev_v1_pancan_conf.json

```diff
@@ -0,0 +1,90 @@
+{
+  "analyticsToken": "UA-108484823-1",
```

Comment:
Please revert it back, this file is extra

Reply:
**Do not revert this. Please do not touch this branch**